listen.js

```diff
@@ -0,0 +1,5 @@
+const app = require("./app");
```

Comment:
Don't need this file to complete this ticket, so shouldn't be included in the PR